frontend/src/components/activitiesPanel.js

```diff
@@ -1,16 +1,37 @@
+import React, { useEffect, useState, useContext } from "react";
 import Container from "react-bootstrap/Container";
 import Card from "react-bootstrap/Card";
+
+import { UserContext } from "../utils/userContext";
 import Activity from "./activity";
+import * as activityService from "../services/activityService";
+
+const ActivitiesPanel = ({ otherUserID }) => {
+  const [activities, setActivities] = useState();
+  const { user } = useContext(UserContext);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const { data } = await activityService.getActivities(
+        otherUserID ? otherUserID : user.id
+      );
+      setActivities(data);
+    };
+    fetchData();
+  }, [user.id]);
```

Comment:
I think no need to put user id since it will initially load the activity data at first visit only. Unless you have select user with activity on the same page